includes/server/llms-rest-server-functions.php

```diff
@@ -80,14 +80,125 @@ function llms_rest_server_error( $message = '' ) {
 }
 
 /**
- * Validate submitted array of integers is an array of real real user ids.
+ * Validate submitted array of integers is an array of real user ids
  *
  * @since 1.0.0-beta.9
  *
  * @param array $instructors Array of instructors id.
- *
  * @return boolean
  */
 function llms_validate_instructors( $instructors ) {
 	return ! empty( $instructors ) ? count( array_filter( array_map( 'get_userdata', $instructors ) ) ) === count( $instructors ) : false;
 }
+
+/**
+ * Validate strict positive integer number
+ *
+ * @since [version]
+ *
+ * @param integer $number Integer number to validate.
+ * @return boolean
+ */
+function llms_rest_validate_strictly_positive_int( $number ) {
+	return llms_rest_validate_positive_int( $number, false );
+}
+
+/**
+ * Validate positive integer number including zero
+ *
+ * @since [version]
+ *
+ * @param integer $number Integer number to validate.
+ * @return boolean
+ */
+function llms_rest_validate_positive_int_w_zero( $number ) {
+	return llms_rest_validate_positive_int( $number );
+}
+
+
+/**
+ * Validate positive integer number
+ *
+ * @since [version]
+ *
+ * @param integer $number Integer number to validate.
+ * @param boolean $include_zero Optional. Whether or not 0 is included. Default is `true`.
+ * @return boolean
+ */
+function llms_rest_validate_positive_int( $number, $include_zero = true ) {
+	return false !== filter_var(
+		$number,
+		FILTER_VALIDATE_INT,
+		array(
+			'options' => array(
+				'min_range' => $include_zero ? 0 : 1,
+			),
+		)
+	);
+}
+
+/**
+ * Validate strict positive float number
+ *
+ * @since [version]
+ *
+ * @param integer $number Float number to validate.
+ * @return boolean
+ */
+function llms_rest_validate_strictly_positive_float( $number ) {
+	return llms_rest_validate_positive_float( $number, false );
+}
+
+/**
+ * Validate strict positive float number including zero
+ *
+ * @since [version]
+ *
+ * @param integer $number Float number to validate.
+ * @return boolean
+ */
+function llms_rest_validate_positive_float_w_zero( $number ) {
+	return llms_rest_validate_positive_float( $number );
+}
+
+/**
+ * Validate strict positive float number
+ *
+ * @since [versoin]
+ *
+ * @param integer $number Float number to validate.
+ * @param boolean $include_zero Optional. Whether or not 0 is included. Default is `true`.
+ * @return boolean
+ */
+function llms_rest_validate_positive_float( $number, $include_zero = true ) {
+	// min_range and max_range options for FILTER_VALIDATE_FLOAT are only available since PHP 7.4.
```

Comment:
One day we can find this note and make an update to use the filter

```suggestion
	// @TODO min_range and max_range options for FILTER_VALIDATE_FLOAT are only available since PHP 7.4.
```